library/Vanilla/EmbeddedContent/Factories/VimeoEmbedFactory.php

```diff
@@ -43,7 +43,7 @@ protected function getSupportedDomains(): array {
      * @inheritdoc
      */
     protected function getSupportedPathRegex(string $domain): string {
-        return "`^/?\d+(\?|$)`";
+        return "`^(/[a-z,[0-9]+){1,2}$`";
```

Comment:
I _believe_ it has to be an all-numeric first segment and then an alphanumeric second segment. This would mean we shouldn't be allowing "," or "[" for either segment.

How about the following?

```suggestion
        return "`/\d+(/[a-z0-9]+)?$`";
```

Reply:
don't we need to escape slashes? and I would keep `^` start line check
`^(\/\d+)(\/[a-z0-9]+)?$`

Reply:
> don't we need to escape slashes?

We're using the back tick (`) as the delimiter here, so escaping the slashes is not necessary.